Assets/Plugins/Source/Editor/Build/BuildRunner.cs

```diff
@@ -20,6 +20,8 @@
  * SOFTWARE.
  */
 
+#if !EOS_DISABLE
+
 namespace PlayEveryWare.EpicOnlineServices.Editor.Build
```

Comment:
Add tasking to address any potential issues that may arise when applied to other platforms.

Reply:
Tasking has been added - marking conversation as resolved.